### ISSUE TITLE
About Pascal VOC

### ISSUE THREAD

**Issue by Serge-weihao** [Jia-Research-Lab/PFENet#12]
Does your code read Pascal VOC gt mask in color lkike this?
![image](https://user-images.githubusercontent.com/40890323/95659180-c5940200-0b51-11eb-9c7a-4a6cc09c0f66.png)

**Comment by tianzhuotao** [comment]
@Serge-weihao Yes, the boundary region should be labeled as 255 (ignore during training and testing).

**Comment by Serge-weihao** [comment]
> @Serge-weihao Yes, the boundary region should be labeled as 255 (ignore during training and testing).

        label = cv2.imread(label_path, cv2.IMREAD_GRAYSCALE)  
this code may not get 0 to 20 on a color label
![image](https://user-images.githubusercontent.com/40890323/95708211-46581880-0c8e-11eb-9806-c57db1731220.png)

**Comment by Serge-weihao** [comment]
I convet .mat to .png on SBD datasets,  and the values of png file is 0 to 20. Can your code reads these  two types of png file together?

**Comment by tianzhuotao** [comment]
@Serge-weihao Our code only supports .png labels currently. Of course, you can check the type of the input label file to use different data loaders for .mat and .png files respectively. As long as the processed ground truth contains 0-20 labels + 255 (ignore label), the algorithm should work well.

If you have further troubles/questions in converting the data, you can send me an email and I can provide the .zip file of pre-processed labels to you directly. 

Thank you.

**Comment by Serge-weihao** [comment]
I found that using Image.open in 'p' mode to load gt png file can solve both colorful png in voc12 and the pngs(0-20) converted from SBD

**Comment by tianzhuotao** [comment]
@Serge-weihao Once you have done the data conversion, you can try with the pre-trained model to check whether your performance can match with ours :)

Thank you~

**Comment by Serge-weihao** [comment]
do your 1-shot and 5-shot trained models share the same weighted?  
![image](https://user-images.githubusercontent.com/40890323/96119129-17a4a100-0f1f-11eb-9df8-9fd4aaa078ad.png)
![image](https://user-images.githubusercontent.com/40890323/96119489-a44f5f00-0f1f-11eb-8c0a-ab0f5f4c5ace.png)
are my settings wrong?
And what's your current email addr?

**Comment by tianzhuotao** [comment]
@Serge-weihao You can find my email address in the paper. Also, I will provide a link in this issue recently.
